python/pyspark/sql/dataframe.py

```diff
@@ -1207,8 +1222,11 @@ def approxQuantile(self, col, probabilities, relativeError):
             raise ValueError("relativeError should be numerical (float, int, long) >= 0.")
         relativeError = float(relativeError)
 
-        jaq = self._jdf.stat().approxQuantile(col, probabilities, relativeError)
-        return list(jaq)
+        jaqs = self._jdf.stat().approxQuantileMultiCols(col, probabilities, relativeError)
```

Comment:
You call the mult-columns version anyway no matter your input is a string or a list. If you like to do so, then the one column version of python api is not needed anymore.